### ISSUE TITLE
Gitcoin grant

### ISSUE THREAD

**Issue by thomasmetta** [Developer-DAO/developerdao.com#64]
Let's apply for Gitcoin grant https://gitcoin.co/grants/quickstart

**Comment by thomasmetta** [comment]
There's a community and NFT Grant and i think we can apply to both https://gitcoin.co/grants/

**Comment by with-heart** [comment]
Hi @thomasmetta! No pressure here, just wondering if there's an update or if there's anything we can do to help!

**Comment by thomasmetta** [comment]
Hey, sorry I have been MIA this week lol. I will have something draft by tonight/tomorrow

**Comment by thomasmetta** [comment]
Draft

What is DeveloperDAO
The project was started as an experiment by Nader Dabit to see what can be achieved when we bring together talented and like-minded people who are interested in learning about and building on Web3.

The DAO operates inside our discord server and over 240 active users. To gain access to this server you need to own a Devs for Revolution NFT. The contract for purchasing one can be found on etherscan.io. Our community is composed of developers from different backgrounds, we even have one of the core React team member attended our meeting

Community related notes: https://github.com/Developer-DAO/community

Initial deployment of DeveloperDAO NFT: 
https://etherscan.io/token/0x25ed58c027921e14d86380ea2646e3a1b5c55a8b

We have 2 main projects we are working on

DeveloperDAO website:
https://github.com/Developer-DAO/developer-dao

This is the public DeveloperDAO site repo. At the moment it allows members to view their genesis NFT by entering in its ID. As the community takes shape we will decide what this website becomes and build it as a collective.

There’s over 25 active developers working on sites and shipping features everyday

Derivative: https://github.com/Developer-DAO/pixel-avatars
Derivatives of the DeveloperDAO genesis NFTs; similar to how Loot has its own derivatives projects

With your support, we hope to accomplish the following:

- Service cost for discourse. Lot’s of DAO is organized discussion around proposals by using Discourse

- Gas fee for the derivative project smart contract development. Gas fees are expensive right now and we need funding in order to deploy the contract

- Bounty to reward developers who worked on features

- Build the largest community where developers can learn more about web3

- Produce educational material to onboard more web2 users and help them transition to web3

**Comment by gjsyme** [comment]
In the "With your support" section -> Server cost for discourse "Lots" should be used vice "Lot's"

**Comment by Dhaiwat10** [comment]
> - Bounty to reward developers who worked on features

I think we can reword this as:

- Bounties to reward content creators who have produced educational content & developers who have worked on features.

**Comment by Dhaiwat10** [comment]
And maybe put the last point somewhere higher, before this point ^

**Comment by dabit3** [comment]
Looks really great, @thomasmetta. Only minor suggestion I have is `to onboard more web2 users` -> `to onboard more web2 users and developers`

**Comment by nwthomas** [comment]
This looks amazing! Do we want to include a link to the Etherscan contract where we reference it?

`"...purchasing one can be found on etherscan.io"` -> `"...purchasing one can be found on [etherscan.io](https://etherscan.io/token/0x25ed58c027921e14d86380ea2646e3a1b5c55a8b#writeContract)"`

Here's a few more small edits (syntax highlighting are changes/additions):
1. "Our community is composed of developers from different backgrounds, `and` we even have one of the core React team member attended our meeting`.`"
2. "We have 2 main projects we are working on`:`"
3. "Social:" -> Make this a bold header symbolizing a new section

Should we move this to a markdown file to more easily submit PRs against the draft?

**Comment by PatrickAlphaC** [comment]
> Bounties to reward content creators who have produced educational content & developers who have worked on features

Woah we could actually get PAID for the content we make?!
I'm on board.

**Comment by ryancharris** [comment]
> over 240 active users

@thomasmetta How do we define an active user? Also wondering if it's worth mentioning how many users have minted NFTs to join since I believe that number is higher 🤔

**Comment by thomasmetta** [comment]
@nwthomas Great suggestion. The Gitcoin form was just a simple text editor so didn't think of markup. Will make a PR for the final draft

@ryancharris Make sense, there's over 1,585 DEV NFT unique holder and 3,139 minted. The `240` number was the number of people in the discord. Active in a way that they went out of way to verify through collabland to participate in discord. I will include all those 3 numbers

**Comment by kempsterrrr** [comment]
@thomasmetta great effort getting this going, think it all reads great. Maybe worth mentioning the intention to create a roadmap for dev's to get into web3, similar to [roadmap.sh](https://roadmap.sh/) as a public good, although no one has taken that on yet...

I _think_ the number of people who have successfully verified via Collab Land is currently 929. See the image below showing number of people with the `dev` role which I'm _pretty sure_ is the role assigned once verified.

<img width="730" alt="Screenshot 2021-09-27 at 20 01 31" src="https://user-images.githubusercontent.com/9025997/134969446-d3f04199-4954-4c45-ac77-49033cd38219.png">

Could confirmed this by asking @conner#0063 in discord

**Comment by dabit3** [comment]
Update: I met with Gitcoin a few days ago about something else, but we dove deeply into the topic of they DAO and they are highly interested in collaborating in some way, it seems like the incentives are aligned perfectly. Thanks for everyone's work here so far.

**Comment by Dhaiwat10** [comment]
That's some great news @dabit3 🙏
